publish-to-maven-central/SDK4Mvn.aggr

```diff
@@ -2,10 +2,10 @@
 <aggregator:Aggregation xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:aggregator="http://www.eclipse.org/cbi/p2repo/2011/aggregator/1.1.0" label="SDK4Mvn" packedStrategy="UNPACK_AS_SIBLING" type="R" mavenResult="true" versionFormat="MavenRelease" includeSources="true">
   <validationSets label="main">
     <contributions label="sdk">
-      <repositories enabled="false" location="/home/data/httpd/download.eclipse.org/eclipse/updates/4.27-I-build"/>
+      <repositories enabled="false" location="/home/data/httpd/download.eclipse.org/eclipse/updates/4.27-I202303111800"/>
```

Comment:
This does not exist:

https://download.eclipse.org/eclipse/updates/4.27-I202303111800

Eventually there will be a repo like this:

https://download.eclipse.org/eclipse/updates/4.26/R-4.26-202211231800/

And soon there will be some build nested in 

https://download.eclipse.org/eclipse/updates/4.27-I-builds/

e.g.,

https://download.eclipse.org/eclipse/updates/4.27-I-builds/I20230228-0600/

that will be the declared final 4.27 build.

But we don't know that yet.